tests/Pipfile

```diff
@@ -7,18 +7,18 @@ verify_ssl = true
 
 [packages]
 decorator = "==4.3.2"
-elasticsearch = "==6.3.1"
+elasticsearch = ">=6.3.1"
```

Comment:
why changing this?

Reply:
Without this change the list of pinned dependencies cannot be resolved, presumably since this specific, old version of elasticsearch relies on an older (vulnerable) version of urllib